src/libs/actions/SmallDevice.js

```diff
@@ -0,0 +1,13 @@
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../ONYXKEYS';
+
+/**
+ * Is small device
+ */
+function isSmallDevice(isSmall) {
+    Onyx.set(ONYXKEYS.IS_SMALL_DEVICE, isSmall);
```

Comment:
It would be better to just use `Dimensions` wherever you need to know about the small screen width for now. Maybe at some point we can create a single source of truth for the screen width or whether we are below the responsive breakpoint. I don't think this is the right time for that.